bootstrap/grammar.go

```diff
@@ -51,9 +52,10 @@ named   -> (IDENT op="=")? atom;
 quant   -> op=/{[?*+]}
          | "{" min=INT? "," max=INT? "}"
          | op=/{<:|:>?} opt_leading=","? named opt_trailing=","?;
-atom    -> IDENT | STR | RE | "(" term ")" | "(" ")";
+atom    -> IDENT | STR | RE | REF | "(" term ")" | "(" ")";
 
 // Terminals
+REF		-> ‵\‵ IDENT;
```

Comment:
nit: Better in the same of order of appearance as in the non-terminals section. On the same basis, if you feel like it, move COMMENT to the front.

Reply:
k